ddtrace/settings/matching.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
We will want this to be a private module, maybe something in `ddtrace.internal`, or just renaming as `_matching.py` ?